ionoscloud/resource_dbaas_pgsql_user.go

```diff
@@ -0,0 +1,180 @@
+package ionoscloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	pgsql "github.com/ionos-cloud/sdk-go-dbaas-postgres"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/dbaas"
+	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils"
+	"strings"
+)
+
+func resourceDbaasPgSqlUser() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDbaasPgSqlUserCreate,
+		UpdateContext: resourceDbaasPgSqlUserUpdate,
+		ReadContext:   resourceDbaasPgSqlUserRead,
+		DeleteContext: resourceDbaaSPgSqlUserDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceDbaasPgSqlUserImporter,
+		},
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
+				Type:             schema.TypeString,
+				Required:         true,
+				ForceNew:         true,
```

Comment:
no need for forcenew

Reply:
@cristiGuranIonos based on what we discussed, we will leave this as it is, you can mark this as solved.